tests/dash/conftest.py

```diff
@@ -202,3 +208,17 @@ def apply_direct_configs(dash_outbound_configs, apply_config):
     del dash_outbound_configs[VNET2_NAME]
 
     apply_config(dash_outbound_configs)
+
+
+@pytest.fixture(scope="module", autouse=True)
+def setup_gnmi_server(duthosts, rand_one_dut_hostname, localhost):
+    if ENABLE_GNMI_API is False:
+        yield
+        return
+
+    duthost = duthosts[rand_one_dut_hostname]
+    duthost.shell("docker exec gnmi rm /usr/local/yang-models/sonic-dash.yang", module_ignore_errors=True)
```

Comment:
>rm [](http://example.com/codeflow?start=36&length=2)

Why remove the file? On host or in gnmi container?

Reply:
Current image will run yang validation, so I have to remove yang file to disable yang validation.
I will upgrade image to remove yang validation.